docs/contracts/errors.rst

```diff
@@ -79,4 +79,4 @@ of the built-in type ``Panic(uint256)``.
     of inner calls is propagated back through the chain of external calls
     by default. This means that an inner call
     can "forge" revert data that looks like it could have come from the
-    contract that called it.
\ No newline at end of file
+    contract that called it.
```

Comment:
Is there actually a difference here? If it was a newline github would show that but it does not so I wonder what's going on here.

Reply:
Yeah it's weird I didn't catch out what's changed here. I think in this script I just highlighted Solidity.